### DISCUSSION TITLE
可视化页面搭建问题请教

### DISCUSSION THREAD

**Discussion by YangWenLong123** [ly525/luban-h5#227] [Q&A] [answered]
去做一个这样的需求，基本模块都是定制化开发，每个模块的数据是怎么样去设计保存不被其它模块污染，如何去构建页面，搭建完成后如何把页面存储服务器？

**Comment by ly525** [comment, marked as answer]
你先说下你自己的理解吧
1. 你说的模块是指什么？
2. 你说的构建页面是什么意思？
3. `把页面存储至服务器`：一般是json 数据

建议你先看看可视化搭建的基础内容，不要做伸手党，基础调研还是要做的！
